spec/features/admin/merchants/index_spec.rb

```diff
@@ -109,16 +109,108 @@
+        InvoiceItem.create!(item_id: item2.id, invoice_id: invoice2.id, quantity: 6, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item1.id, invoice_id: invoice1.id, quantity: 4, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item6.id, invoice_id: invoice7.id, quantity: 1, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item5.id, invoice_id: invoice8.id, quantity: 1, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item4.id, invoice_id: invoice9.id, quantity: 1, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item3.id, invoice_id: invoice10.id, quantity: 1, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item2.id, invoice_id: invoice11.id, quantity: 1, unit_price: 100, status: 2)
+        InvoiceItem.create!(item_id: item1.id, invoice_id: invoice12.id, quantity: 1, unit_price: 100, status: 2)
+        invoice1.transactions.create!(result: 1)
+        invoice2.transactions.create!(result: 0)
+        invoice3.transactions.create!(result: 0)
+        invoice3.transactions.create!(result: 1)
+        invoice4.transactions.create!(result: 1)
+        invoice5.transactions.create!(result: 1)
+        invoice6.transactions.create!(result: 1)
+        invoice7.transactions.create!(result: 1)
+        invoice8.transactions.create!(result: 1)
+        invoice9.transactions.create!(result: 1)
+        invoice10.transactions.create!(result: 1)
+        invoice11.transactions.create!(result: 1)
+        invoice12.transactions.create!(result: 1)
+        
+        visit admin_merchants_path
+
+        within("#top_5-#{merchant1.id}") do
+          expect(page).to have_content(merchant1.best_day)
+        end
+
+        within("#top_5-#{merchant3.id}") do
+          expect(page).to have_content(merchant3.best_day)
+        end
+
+        within("#top_5-#{merchant4.id}") do
+          expect(page).to have_content(merchant4.best_day)
+        end
+
+        within("#top_5-#{merchant5.id}") do
+          expect(page).to have_content(merchant5.best_day)
+        end
+
+        within("#top_5-#{merchant6.id}") do
+          expect(page).to have_content(merchant6.best_day)
+        end
+        # within("#top_5-#{merchant1.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant1))
+        # end
+
+        # within("#top_5-#{merchant3.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant3))
+        # end
+
+        # within("#top_5-#{merchant4.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant4))
+        # end
+
+        # within("#top_5-#{merchant5.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant5))
+        # end
+
+        # within("#top_5-#{merchant6.id}") do
+        #   expect(page).to have_content(Merchant.best_day(merchant6))
+        # end
```

Comment:
@rjdelaguna I wanted to check in before removing these commented out lines. Let me know if I can remove them and I will commit it in this PR.